sql/planbuilder/project.go

```diff
@@ -88,7 +88,7 @@ func (b *Builder) analyzeSelectList(inScope, outScope *scope, selectExprs ast.Se
 			outScope.addColumn(scopeColumn{tableId: inScope.tables[e.Table()], table: e.Table(), db: e.Database(), col: e.Name(), scalar: e, typ: e.Type(), nullable: e.IsNullable(), id: id})
 		case *expression.Star:
 			tableName := strings.ToLower(e.Table)
-			if tableName == "" && len(inScope.cols) == 0 {
+			if tableName == "" && len(inScope.cols) == 1 && inScope.cols[0].col == "" {
```

Comment:
This is only to support `dual`, right? Might be worth a helper function, like `isDualColumn()` or even testing if `cols[0].tableName == 'dual'` or something, just to make it super obvious to future people reading this code in the future.

Reply:
I can add a "dual" table name to this column, shouldn't hurt anything